src/core/tasks.py

```diff
@@ -93,19 +93,20 @@ def get_validators(self):
                 yield from component.get_validators()
 
     def get_status_message(self):
-        message = {
-            "type": "success",
-            "template": self.status_message_template,
-        }
+        message = {}
+        if self.status_message_template:
+            message["type"] = "success"
+            message["template"] = self.status_message_template
         return message
 
     def as_dict(self):
         screen = super().as_dict()
         screen["components"] = self.get_flow_components()
 
         if self.show_status_message:
```

Comment:
Doesn't that just make this flag invalid as you can just set status_message to None?

Reply:
You might what to check where this flag is used before removing it though.